### PR TITLE
ci: Run Macrobenchmarks nightly

### PR DESCRIPTION
**Pull request by PROFeNoM**
### Description

Run macrobenchmarks nightly (master). 

More specifically, the PR introduces a `NIGHTLY_BENCHMARKS` to specifically allow the benchmarks to be run with the added corresponding [scheduled pipeline](https://gitlab.ddbuild.io/DataDog/apm-reliability/dd-trace-php/-/pipeline_schedules).

### Reviewer checklist
- [ ] Test coverage seems ok.
- [ ] Appropriate labels assigned.

### CODE REVIEW

**Review by ddyurchenko — APPROVED** [review]

**Review by bwoebi — APPROVED** [review]
Oh, I suppose it needs an ack from a maintainer too. Quickly acking here :-D

**Pull request merged**
